src/main/java/com/github/manolo8/darkbot/gui/players/PlayerManager.java

```diff
@@ -58,6 +60,20 @@ private void addPlayerByUsername() {
             if (name == null) return;
             editor.main.config.UNRESOLVED.add(new UnresolvedPlayer(name));
         }
+
+        private void addSelectedPlayer() {
+            if (editor.getPlayerInfoList().getSelectedValuesList().isEmpty()) {
```

Comment:
```suggestion
            if (editor.getPlayerInfoList().isSelectionEmpty()) {
```

Avoids having to create the list of values when you can just check for empty